tools/go.sum

```diff
@@ -116,6 +116,8 @@ github.com/ashanbrown/makezero v0.0.0-20210520155254-b6261585ddde/go.mod h1:oG9D
 github.com/aws/aws-sdk-go v1.23.20/go.mod h1:KmX6BPdI08NWTb3/sm4ZGu5ShLoqVDhKgpiN924inxo=
 github.com/aws/aws-sdk-go v1.25.37/go.mod h1:KmX6BPdI08NWTb3/sm4ZGu5ShLoqVDhKgpiN924inxo=
 github.com/aws/aws-sdk-go v1.36.30/go.mod h1:hcU610XS61/+aQV88ixoOzUoG7v3b31pl2zKMmprdro=
+github.com/benbjohnson/clock v1.1.0 h1:Q92kusRqC1XV2MjkWETPvjJVqKetz1OzxZB7mHJLju8=
```

Comment:
Is there a reason we're adding this in?

Reply:
This is just go.sum dependency closure stuff

Reply:
Looks like zap 1.19
```
go mod graph | grep clock
go.uber.org/zap@v1.19.0 github.com/benbjohnson/clock@v1.1.0
github.com/spf13/viper@v1.7.0 github.com/jonboulle/clockwork@v0.1.0
github.com/spf13/viper@v1.4.0 github.com/jonboulle/clockwork@v0.1.0
go.etcd.io/etcd@v0.0.0-20191023171146-3cf2f69b5738 github.com/jonboulle/clockwork@v0.1.0
github.com/google/certificate-transparency-go@v1.1.1 github.com/jonboulle/clockwork@v0.2.0
github.com/spf13/viper@v1.7.1 github.com/jonboulle/clockwork@v0.1.0
go.etcd.io/etcd@v0.0.0-20200513171258-e048e166ab9c github.com/jonboulle/clockwork@v0.1.0
```